### DISCUSSION TITLE
Experiment doesn't work on psiTurk/mTurk (but works fine as .html)

### DISCUSSION THREAD

**Discussion by fspinar** [jspsych/jsPsych#2093] [Q&A]
Hi all,

I am trying to run my experiment by using jsPsych. The code works fine when I run it as an HTML code separate from psiTurk, but when I try to run it on mTurk sandbox or in the debug mode of psiTurk, I only get a blank screen.

Could you help me through this ? I really don't see what is wrong.

Thanks a lot.

**Comment by becky-gilbert** [comment]
Hi @fspinar, it'll be hard to figure out what the problem is here without more information. Do you get any errors in the browser's JavaScript console when you see the blank screen? Also, have you tried reaching out to psiTurk support?

**Reply by fspinar** [reply]
Hello again @becky-gilbert 
I looked into the web browser's log file, and I got "uncaught reference error, jsPsych is not defined". This happens when I first attempted to use jsPsych in my code. This is despite the fact that I have jsPsych under : jsPsych under ~/static/js/jspych. Any idea as to why jsPsych is not recognized ?

PS : I asked the same question to psiturk support, too, but I am asking here also becasuse I really need help...
 
Thanks a lot.

**Reply by becky-gilbert** [reply]
Hi @fspinar, it sounds like maybe the core jsPsych file isn't loading properly, but this is still really tricky to debug without more information. Are you seeing any other errors in the JavaScript console, or just the "uncaught reference" error? Could you post a live link to your experiment so that I can see the errors and code? If not, could you post your code here (maybe just a minimal hello-world-type example that produces the same problem, if your actual experiment is complicated)?

**Reply by fspinar** [reply]
@becky-gilbert I also think that the core jsPsych isn't loading properly
I can send you screenshot of the javascript console ? 

A question : In order to load jspysch, do I need to anything other than including jspych under static/js/psych folder ?

![Screenshot (182)](https://user-images.githubusercontent.com/39970773/132929612-57626513-ab1c-4106-a057-94a20adcdc5e.png)

If this doesn't work I can also send the link in the debug mode. The experiment is complicated but I think the only issue is to load jspsych correctly. 

Thanks again.

**Reply by jodeleeuw** [reply]
Hi @fspinar ,

You probably need to include a `<script>` tag to load jsPsych in the correct html page.

**Comment by deargle** [comment]
:wave: from psiturk support. https://github.com/NYUCCL/psiTurk/issues/525

**Comment by jodeleeuw** [comment]
I'm going to shift this to [discussions](https://github.com/jspsych/jsPsych/discussions) since it is a support request.
